jiant/utils/data_loaders.py

```diff
@@ -41,6 +41,9 @@ def load_span_data(tokenizer_name, file_name, label_fn=None, has_labels=True):
     rows = rows.apply(lambda x: realign_spans(x, tokenizer_name), axis=1)
     if has_labels is False:
         rows["label"] = False
+    else:
+        if label_fn is not None:
+            rows["label"] = rows["label"].apply(lambda x: label_fn(x))
```

Comment:
`lambda x: f(x)` is identical to `f(x)`—replace.

I can't tell if this is Python apply (which you could replace with just `label_fn(rows ...` or Pandas apply, which does seem to do something.

Reply:
I think it's pandas, but you can still replace with just f(x)

Reply:
You need to have the lambda x since otherwise x is not defined in f(x)